ionhash/__init__.py

```diff
@@ -15,7 +15,7 @@
 
 from amazon.ion.core import ION_STREAM_END_EVENT
 from amazon.ion.simple_types import _IonNature
-from amazon.ion.simpleion import _dump
+from amazon.ion.simpleion import _dump, _FROM_TYPE
```

Comment:
As discussed offline, it would be nice to be able to use the public version of the `dump` method. I've opened amzn/ion-python#112 to explore whether that's worthwhile.